dspace-api/src/main/java/org/dspace/scripts/handler/DSpaceRunnableHandler.java

```diff
@@ -114,4 +116,6 @@ public interface DSpaceRunnableHandler {
      */
     public void writeFilestream(Context context, String fileName, InputStream inputStream, String type)
         throws IOException, SQLException, AuthorizeException;
+
+    public List<UUID> getSpecialGroups();
```

Comment:
Can we add JavaDocs to this to describe where these special groups are coming from?  It's not immediately obvious if you look at this class because `DSpaceRunnableHandler` has no internal special groups.  So, this method is not just a basic getter which returns a property from `DSpaceRunnableHandler`.